compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/FirDefaultErrorMessages.kt

```diff
@@ -778,6 +779,11 @@ class FirDefaultErrorMessages : DefaultErrorMessages.Extension {
             )
             map.put(NON_FINAL_MEMBER_IN_FINAL_CLASS, "'open' has no effect in a final class")
             map.put(NON_FINAL_MEMBER_IN_OBJECT, "'open' has no effect in an object")
+            map.put(
+                VIRTUAL_MEMBER_HIDDEN, "''{0}'' hides member of supertype ''{2}'' and needs ''override'' modifier", DECLARATION_NAME,
```

Comment:
Looks like you don't need the 2nd parameter ("overridden") and this can just be `{0}` and `{1}`